### ISSUE TITLE
RFC: Chat UI - message time position

### ISSUE THREAD

**Issue by lkarakoleva** [telerik/kendo-ux#6]
Currently in chat UI implementation, the message time is shown next to a selected message and the message status beneath it:
![untitled-1](https://user-images.githubusercontent.com/16577622/40046081-b02474f4-5834-11e8-9c58-510b420f1de2.png)

On the one hand is it better experience the short message, time to be displayed next to it. On the other hand if the time format is too long, it would be cut out of the chat area.

We could use the same approach as in mobile facebook messenger:
![2](https://user-images.githubusercontent.com/16577622/40046073-ab5012b2-5834-11e8-81c0-7990f9fcc2ce.png)

The timeline of the conversation is center aligned. We have already had timestamp placed at the center of the chat canvas so we could follow the same trend for the single message time - at the center of the chat above the message. 

@tsvetomir @ivanchev @mitagg @joneff @gyoshev

**Comment by tsvetomir** [comment]
The desktop Messenger shows message timestamps in a popup on hover.

![image](https://user-images.githubusercontent.com/247917/40053572-77421478-584a-11e8-8c29-4d6e16068aa9.png)

The visible "timeline timestamps", for a lack of a better term, are displayed only when a significant time elapses between messages - not sure how much exactly.

**Comment by lkarakoleva** [comment]
Should we separate the mobile and desktop message time display? If we keep the same behavior for both mobile and desktop is it better to use the mobile approach, as we do not have "on hover" interaction in mobile.

**Comment by tsvetomir** [comment]
No, the mobile approach is better and more consistent. We'd still show them only for selected messages, right?

**Comment by lkarakoleva** [comment]
We keep the interaction as it is - showing message time only on selection. My suggestion is to change only the position of the time - at the center of the chat above the message.

**Comment by ivanchev** [comment]
The times shown in the center above the message are indeed used to indicate a new "era", for instance a new day. Using the same layout for selected messages might be confusing. Is there a suggestion as to how we should differentiate between the two?

**Comment by lkarakoleva** [comment]
Here there is an idea how to differentiate the message time text and the era text
![1](https://user-images.githubusercontent.com/16577622/40180394-6267ba20-59ef-11e8-96b5-ead8cbf01e55.png)
